presto-spark-base/src/main/java/com/facebook/presto/spark/execution/NativeExecutionProcess.java

```diff
@@ -63,7 +65,7 @@
         implements AutoCloseable
 {
     private static final Logger log = Logger.get(NativeExecutionProcess.class);
-    private static final String NATIVE_EXECUTION_TASK_ERROR_MESSAGE = "Encountered too many errors talking to native process. The process may have crashed or be under too much load.";
+    private static final String NATIVE_EXECUTION_TASK_ERROR_MESSAGE = "Native process launch failed with multiple retries.";
```

Comment:
> multiple retries

Would it be possible to specify the exact number of retries?

Reply:
We specify the duration for the retry (maxErrorDuration) to the RequestErrorTracker and it will retry with certain times with backoff. In the finally error message threw by the RequestErrorTracker, it will contains above message with actual retry count.